Library/BillingInfo.cs

```diff
@@ -15,7 +15,12 @@ public enum CreditCardType : short
             MasterCard,
             AmericanExpress,
             Discover,
-            JCB
+            JCB,
+            Danokrt,
+            Maestro,
+            Forbrugsforeningen,
+            Laser,
+            Unknown

```

Comment:
The name of the value from an API doesn't need to match the enum name. We'll have to sanitize the value somewhere such as the xml reader to remove spaces in the value and then it will match the enum name.